lib/debug/server_dap.rb

```diff
@@ -706,6 +708,7 @@ def process_dap args
 
           path = frame.realpath || frame.path
```

Comment:
nit: we can call `skip_path?` right after this so we don't need to compute `source_name` and throw it away

Reply:
Yes, we can :) changed it